src/stock-adjustment-mobile/adjustment-form.component.jsx

```diff
@@ -99,6 +110,11 @@ const AdjustmentForm = ({ stockAdjustmentCreationService,
                 data={adjustment}
                 dataToDisplay={dataToDisplay}
                 headerToDisplay={headerToDisplay}
+                onClickAction={editProduct}
+            />
+            <Toast 
+                autoDelete={true}
```

Comment:
If you pass no value for a prop it defaults to `true`: https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true

No need to make any changes, but it might be useful for you in the future.